src/main/kotlin/net/lumalyte/lg/utils/NexoItemProvider.kt

```
@@ -0,0 +1,64 @@
+    /**
+     * Attempts to get a Nexo custom item by its item ID (e.g. "lg_nav_info").
+     *
+     * @param itemId The Nexo item identifier (without the "lg_" prefix — pass the full ID).
+     * @return The ItemStack from Nexo, or null if Nexo is unavailable or the ID is not found.
+     */
+    fun getItemStack(itemId: String): ItemStack? {
+        val api = nexoApiClass ?: return null
+        return try {
+            val method = api.getMethod("getItemStack", String::class.java)
+            method.invoke(null, itemId) as? ItemStack
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)